mobile/src/bundles/users/components/new-account-header/new-account-header.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { Text, View } from '~/bundles/common/components/components';
+import {
+    OnboardingScreenName,
+    TextCategory,
+} from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { StepProgressBar } from './step-progress-bar';
+import { StepTrack } from './step-track';
+
+type Properties = {
+    title: (typeof OnboardingScreenName)[keyof typeof OnboardingScreenName];
```

Comment:
Can we use ValueOf in this case ?

Reply:
Yes ... I didn't notice that we have already such type ... 
I will fix it, thank you

Reply:
done